translation/source/site.xml

```diff
@@ -508,6 +508,7 @@
   <string name="endgamePositions">Endgame positions</string>
   <string name="chess960StartPosition">Chess960 start position: %s</string>
   <string name="startPosition">Starting position</string>
+  <string name="startPosFEN">rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR w KQkq - 0 1</string>
```

Comment:
that's not translatable, it doesn't belong in translations. Instead use chessops like here https://github.com/lichess-org/lila/blob/master/ui/puzzle/src/moveTree.ts#L2-L2